website/pages/en/new-chain-integration.mdx

```diff
@@ -1,75 +1,150 @@
-
-New EVM chain integrators may also consider the Firehose-based approach, given the benefits of substreams and its massive parallelized indexing capabilities. Supporting both allows developers to choose between building substreams or subgraphs for the new chain.
-
-> **NOTE**: A Firehose-based integration for EVM chains will still require Indexers to run the chain's archive RPC node to properly index subgraphs. This is due to the Firehose's inability to provide smart contract state typically accessible by the `eth_call` RPC method. (It's worth reminding that eth_calls are [not a good practice for developers](https://thegraph.com/blog/improve-subgraph-performance-reduce-eth-calls/))
-
----
-
-## Testing an EVM JSON-RPC
+#### Testing an EVM JSON-RPC
 
 For Graph Node to be able to ingest data from an EVM chain, the RPC node must expose the following EVM JSON RPC methods:
 
 - `eth_getLogs`
-- `eth_call` \_(for historical blocks, with EIP-1898 - requires archive node):
+- `eth_call` (for historical blocks, with EIP-1898 - requires archive node)
 - `eth_getBlockByNumber`
 - `eth_getBlockByHash`
 - `net_version`
 - `eth_getTransactionReceipt`, in a JSON-RPC batch request
-- _`trace_filter`_ _(optionally required for Graph Node to support call handlers)_
+- `*trace_filter*` *(optionally required for Graph Node to support call handlers)*
 
-### Graph Node Configuration
+### 2. Firehose Integration
 
-**Start by preparing your local environment**
+[Firehose](https://firehose.streamingfast.io/firehose-setup/overview) is a next generation extraction layer. It collects history in flat files and streams in real-time. Firehose technology replaces those polling API calls with a stream of data utilizing a push model that sends data to the indexing node faster. This helps increase the speed of syncing and indexing.
 
-1. [Clone Graph Node](https://github.com/graphprotocol/graph-node)
-2. Modify [this line](https://github.com/graphprotocol/graph-node/blob/master/docker/docker-compose.yml#L22) to include the new network name and the EVM JSON RPC compliant URL
-   > Do not change the env var name itself. It must remain `ethereum` even if the network name is different.
-3. Run an IPFS node or use the one used by The Graph: https://api.thegraph.com/ipfs/
+The primary method to integrate the Firehose into chains is to use an RPC polling strategy. Our polling algorithm will predict when a new block will arrive and increase the rate at which it checks for a new block near that time, making it a very low-latency and efficient solution. For help with the integration and maintenance of the Firehose contact the StreamingFast team. New chains and their integrators will appreciate the [fork awareness](https://substreams.streamingfast.io/documentation/consume/reliability-guarantees) and massive parallelized indexing capabilities that Firehose and Substreams brings to their ecosystem.
```

Comment:
```suggestion
The primary method to integrate the Firehose into chains is to use an RPC polling strategy. Our polling algorithm will predict when a new block will arrive and increase the rate at which it checks for a new block near that time, making it a very low-latency and efficient solution. For help with the integration and maintenance of the Firehose, contact the StreamingFast team. New chains and their integrators will appreciate the [fork awareness](https://substreams.streamingfast.io/documentation/consume/reliability-guarantees) and massive parallelized indexing capabilities that Firehose and Substreams brings to their ecosystem.
```
(also, should "contact the StreamingFast team" be a link?)

Reply:
good call. @Giuliano-1 , you may want to add a link/e-mail/telegram id here, right?

Reply:
Nevermind. I now see it is there.